base/applications/rapps/lang/tr-TR.rc

```diff
@@ -239,7 +240,7 @@ STRINGTABLE
 BEGIN
     IDS_LICENSE_OPENSOURCE "Açık Kaynak"
     IDS_LICENSE_FREEWARE "Ücretsiz"
-    IDS_LICENSE_TRIAL "Deneme/Tanıtım"
+    IDS_LICENSE_TRIAL "Deneme Sürümü"
```

Comment:
The corresponding English translation is `IDS_LICENSE_TRIAL "Trial/Demo"`
so, something similar translated in Türkish would be preferable.

Reply:
I wrote it this way to make it more meaningful in Turkish so there is no mistake on turkish

Reply:
Unresolved also. What Hermes complained about was the missing slash between the 2 words.

Reply:
[IDS_LICENSE_TRIAL](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDS_LICENSE_TRIAL) seems to be a bit controversial, see ru-RU and uk-UA there.

Reply:
Well, maybe those could be improved too, outside of this PR. You can judge that better as a native speaker @binarymaster .

Reply:
Okay, the google translator gave me more insight in scropools change:
![image](https://github.com/user-attachments/assets/6e4affaa-59c4-4c1f-947f-8980884fb751)

He stripped the 2nd term. Is ok for me then.

Reply:
Ok, in this specific tr-TR case I would suggest either `Deneme Sürümü/Tanıtım` or `Deneme/Tanıtım Sürümü`.